articles/tutorials/04_working_with_textures/index.md

```diff
@@ -0,0 +1,364 @@
+
+```cs
+_logo = Content.Load<Texture2D>("images/logo");
+```
+
+The change we made now uses the instance of the `ContentManager` object provided by the `Content` property in the `Game` class. We call the `Load<T>` method, where `T` is the content type that we are loading. The parameter we pass to it is the *asset name* which is a string representation of the path to the content to load, without any extensions. The path used as the *asset name* is also relative to the `Content.RootDirectory` value, which was set to `Content` in the `Game1` constructor above. This means the *asset name* of our logo image located at *Content/images/logo.png* is `images/logo`.
+
+If you run the game now, you'll get the same result as before when we loaded the texture from file. The texture will be loaded in, without error. Before we move on to drawing the texture, let's briefly cover what is actually happening now that we are using the *content pipeline*.
+
+* When you opened the MGCB Editor, the *Content.mgcb* file in your project was loaded inside it. Then in the editor, you added the *logo.png* file. When you saved the changes, these changes were written back to the *Content.mgcb* file.
+* When you told the project to run, it has to perform a build first. One of the NuGet references in a MonoGame project is the *MonoGame.Content.Builder.Tasks* NuGet reference. This adds build tasks that will perform the following:
+  1. Compile the assets defined in the *Content.mgcb* file using the MonoGame Content Builder Tool (MGCB Tool) into *.xnb* files. This compilation process performs the pre-processing of the content.
+  2. Copy the compiled *.xnb* files to your game project's build directory
+* Used the `ContentManager` at runtime to load the compiled asset.
+
+The following image illustrates this workflow:
+
+<figure><img src="./images/content-pipeline-workflow-full.png" alt="Figure 4-6: MonoGame Content Pipeline Workflow"><figcaption><p><strong>Figure 4-6: MonoGame Content Pipeline Workflow</strong></p></figcaption></figure>
+
+While doing all of this may have seemed like more work than just loading from file, it doesn't actually add more work on you as the developer.
+
+* You add the assets using the MGCB Editor instead of just copying them to you *Content* directory.
+* The *MonoGame.Content.Build.Tasks* reference builds and copies the assets to your game project build directory instead of you having to edit the *.csproj* file for each asset.
+* You use the `ContentManager` to load the assets instead of a `FromFile` method call.
+
+And for the same amount of steps, you also get the benefits mentioned before of the assets being pre-processed to an optimized format for the platform(s) your game is targeting. Regardless of which method you use to load assets, the end result gives you a texture loaded and ready to be drawn. Let's move on now to drawing the texture.

```

Comment:
* This is definitely something that needs a few more paragraphs to explain. How is this optimized differently? What's the process behind? Personally, I don't have much knowledge on the topic since I have not used the MGCB that much, but I think it would be useful if it's thoroughly explained somewhere.

Reply:
So i don't repeat a lot of existing documentation, I've added a note section here that links to the content pipeline documentation which goes over the optimizations and benefits

Reply:
Ah! Fair enough!